pkgs/by-name/op/openfga/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub, go }:
+
+buildGoModule rec {
+  pname = "openfga";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = "openfga";
+    repo = pname;
```

```suggestion
    repo = "openfga";
```

using pname like this is discouraged.